package 线程安全181126;

/*在线程的run()方法上不能使用throws来声明抛出异常，只能在方法中使用try-catch来处理异常
 * 因为子类覆盖弗雷方法的原则，子类不能跑出新的异常
* */


public class SleepDemo {
	public static void main(String[] args) {

		new Person("小A").start();
		new Person("小B").start();
		new Person("小C").start();
	}
}